docs/zh/UserGuide/IoTDB-SQL-Language/DML-Data-Manipulation-Language.md

```diff
@@ -533,15 +586,16 @@ IoTDB 支持在 `select` 字句中执行由**数字常量，时间序列、算
 
 ```sql
 selectClause
-    : SELECT resultColumn (',' resultColumn)*
-    ;
+:
+SELECT resultColumn(',' resultColumn) *
+;
 
 resultColumn
-    : expression (AS ID)?
+: expression (AS ID)?
```

Comment:
Please check the indentation.

Reply:
This is the fault of IDEA markdown auto-check. I have rewritten the docs.